### ISSUE TITLE
Docker build is failing on travis

### ISSUE THREAD

**Issue by robjloranger** [gomods/athens#899]
The new change to the proxy dockerfile is failing when built during CI. It does build fine when done locally.

I'm thinking of moving the call to `date` out on the argument passed to the go linker and storing in a env var or build arg.

I'm working on this.

**Comment by nunix** [comment]
Hi Robbie,
I've made it work by changing the following lines in the Dockerfile:
```Dockerfile
COPY . .

RUN apk add git
RUN GO111MODULE=off CGO_ENABLED=0 go build -ldflags "-X github.com/gomods/athens/pkg/build.version=$VERSION -X github.com/gomods/athens/pkg/build.buildDate=$(date -u +%Y-%m-%d-%H:%M:%S-%Z)" -o /bin/athens-proxy ./cmd/proxy

FROM alpine
```

I switched off go modules for the compilation as it doesn't find it (normal as we are building it no?):
`GO111MODULE=off` and removed the `mod=vendor` after `go build`

Finally, here is my build command from `athens` root directory:
```Bash
cd ~/github.com/athens
docker build -t gomods/proxy . -f ./cmd/proxy/Dockerfile
```

The build succeeds like this, however not sure if it's a workaround or a real fix, let me know :)

**Comment by robjloranger** [comment]
@nunix the build works fine locally, it's failing in the Travis environment for some reason. Could be the `GO111MODULES`, I will check it out. I thought maybe it was cgo related as well.

Reopening as the Travis build is still failing.

**Comment by marpio** [comment]
The error msg:
https://travis-ci.org/gomods/athens/builds/453641688#L1457

**Comment by robjloranger** [comment]
Closed with #903
